drake/systems/plants/test/testSignedDistance.m

```diff
@@ -0,0 +1,49 @@
+function testSignedDistance
+
+visualize = 0;
+options.floating = true;
+options.terrain = RigidBodyFlatTerrain();
+
+s = 'FallingBrick.urdf';
```

Comment:
would be good to test this with one of the models that has zero-radius spheres, too.  (e.g. atlas_minimal_contact, or even the fallingbrickcontactpoints)  I don't expect it to behave well, but want to make sure it doesn't do something that could cause problems.